plenum/server/node.py

```diff
@@ -1905,6 +1909,9 @@ def is_catchup_needed(self) -> bool:
         Check if all requests ordered till last prepared certificate
         Check if last catchup resulted in no txns
         """
+        if not self.view_change_in_progress:
```

Comment:
Looks a bit counterintuitive, it would be nice to have a comment explaining why no catchup is needed if view change is not in progress

Reply:
Done